File: python/hello.py
import json
import os


def handler(event, context):
    if "badHeader" in event["headers"]:
        return {
            "statusCode": 400,
            "body": json.dumps({
                "message": "Bad Request!",
            })
        }

    return {
        "statusCode": 200,
        "body": json.dumps({
            "message": os.environ["HELLO"],
        }),
    }

File: nodejs/hello.js
export async function handler(event) {
  console.log(event);
  if ("badHeader" in event.headers) {
    return {
      statusCode: 400,
      body: JSON.stringify({
        message: "Bad Request!",
      }),
    };
  }

  return {
    statusCode: 200,
    body: JSON.stringify({
      message: process.env.HELLO,
    }),
  };
}
